app/cdap/components/NoRecordScreen/styles.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { makeStyles } from '@material-ui/core';
+
+export const useStyles = makeStyles({
+  noRecordWrapper: {
+    height: '100%',
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    position: 'absolute',
+    top: 0,
+    right: 0,
+    left: 0,
+    bottom: 0,
+    '& *': {
+      fontFamily: "'Noto Sans', sans-serif",
+    },
+  },
+  mainHeaderMessage: {
+    fontSize: 20,
+    color: '#000000',
+    marginTop: 20,
+  },
+  subHeaderMessage: {
+    fontSize: 16,
+    color: '#000000',
+  },
+  innerWrapper: {
+    textAlign: 'center',
+  },
+});
```

Comment:
Please add a newline at the end of the file

Reply:
Also, please use `styled-components` for all new styling

Reply:
Please add a newline at the end of the file - DONE
Also, please use styled-components for all new styling - Sure, We will use styled-components for all new styling going forward